lib/voice/VoiceConnection.js

```diff
@@ -809,10 +958,20 @@ class VoiceConnection extends EventEmitter {
         if(this.ws?.readyState === WebSocket.OPEN) {
             data = JSON.stringify({op: op, d: data});
             this.ws.send(data);
+            console.log("<-", data);
             this.emit("debug", data);
         }
     }
 
+    sendWSBinary(op, data) {
+        if(this.ws?.readyState === WebSocket.OPEN) {
+            this.#wsSendBuffer ??= Buffer.allocUnsafe(1024);
+            data.copy(this.#wsSendBuffer, 1);
+            this.#wsSendBuffer.writeUInt8(op);
+            this.ws.send(this.#wsSendBuffer.subarray(0, 1 + data.length));
```

Comment:
this is going to fail with message sizes from 1kB beyond (if any binary messages ever cross that boundary), might want to add a slow path here or rewrite that block entirely

Reply:
Should I make it so this just allocates more if the #wsSendBuffer length isn't large enough?

Reply:
I was thinking more along the lines of allocating a temporary buffer (bypassing #wsSendBuffer) when the binary data being sent is too large (think beyond a few kB), which should be a good enough tradeoff between speed and memory usage – having a 50 MB buffer dangling when someone tries to send a hypothetical 50 MB payload through it doesn't sound good